db/seeds.rb

```diff
@@ -37,9 +37,8 @@
   section: topic,
   type: GeneralContent,
   state: :published
-}) do |node|
-  node.revise!(content_body: 'lorem ipsum').apply!
-end
+})
+node2.revise!(content_body: 'lorem ipsum').apply!
```

Comment:
@micapam There was an error in the seeds. I think I understand what you are saying. Will address.